### PR TITLE
Enhancement: Add admin PDF conversion test page

### PR DESCRIPTION
**Pull request by tomdickman**

### CODE REVIEW

File: test.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Page for testing PDF conversion.
```
Comment (dmitriim) [inline test.php:18]: can you please update README with the test page details?

Comment (tomdickman) [reply test.php:18]: README updated with instructions for testing conversions

File: README.md [hunk 2]
```diff
@@ -66,6 +67,19 @@ $file = $converter->get_converted_moodle_url_pdf('course1337.pdf');
...
+
+In order to test how a URL will be converted and see the outcome, you can utilise the `/admin/tool/pdfpages/test.php` page in your browser.
+This will utilise the configured converter on the server side to carry out the conversion, creating the converted file in the Moodle file system and then serve up the PDF to the browser.
+
+This page takes the following query parameters:
+- url: (required) the ASCII encoded target URL
+- converter: (optional) the converter name to use
+- filename: (optional) the filename to give the converted PDF
+- options: (optional) a JSON encoded string of converter options (see the relevant converter's docs for more information)
+
```
Comment (dmitriim) [inline README.md:80]: maybe add an example of full URL with all possible params?

Comment (tomdickman) [reply README.md:80]: Added full URL example

**Pull request merged**